R/automation.R

```diff
@@ -175,6 +242,10 @@ string_is_file_like <- function(x) {
         (grepl("\\.[[:alnum:]]+$", x) || file.exists(x))
 }
 
+wrapView <- function(value, ...) {
```

Comment:
Weird, almost wrote that I didn't think you'd need this but looks like scrunch has it too. Can you move this function to `shoji.R`, next to the other `wrap*` functions?

Reply:
Silently ignores `...`?

Reply:
Okay, this may not actually matter, but please change to look more like the `wrapEntity()` function:

```
wrapView <- function(..., value = list(...)) {
    list(element = "shoji:view", value = value)
}
```

Then because you're passing the value as a named parameter already, it will still send a string, but if parameters are added and `value` is allowed to be an object instead of a string it'll be easier to support.